code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -496,44 +490,42 @@
-				default_damage(M)
-				return
-			injury_target.droplimb()
-			to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! Oh god something's missing!</span>")
-	else //fracture
-		if(ismachineperson(M))
-			M.emp_act(EMP_HEAVY)
-			M.adjustBrainLoss(30)
-			to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like some of your components are loose!</span>")
-
-		else if(isslimeperson(M))
-			injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
-			if(!injury_target)
-				default_damage(M)
-				return
-			injury_target.cause_internal_bleeding()
-
-			injury_target = M.get_organ(BODY_ZONE_CHEST)
-			injury_target.cause_internal_bleeding()
-			to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like your inner membrane has been punctured!</span>")
-
-		if(prob(25))
-			injury_target = M.get_organ(BODY_ZONE_CHEST)
-			injury_target.fracture()
-		else
-			injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_R_LEG, BODY_ZONE_R_LEG))
-			if(!injury_target)
-				default_damage(M)
-				return
-			injury_target.fracture()
+	if(prob(20)) //See if they're !!!lucky!!! enough to just chop a hand or foot off first
+		injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+		if(!injury_target)
+			return
+		default_damage(M)
+		injury_target.droplimb()
+		to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! Oh god, something's missing!</span>")
+		return
+		//Species specific punishments first
+	if(ismachineperson(M))
+		M.emp_act(EMP_HEAVY)
+		M.adjustBrainLoss(30)
+		to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like some of your components are loose!</span>")
+		return
+	if(isslimeperson(M))
+		injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+		if(!injury_target)
+			return
+		default_damage(M)
+		injury_target.cause_internal_bleeding()
+		injury_target = M.get_organ(BODY_ZONE_CHEST)
+		injury_target.cause_internal_bleeding()
+		to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like your inner membrane has been punctured!</span>")
+		return
+	if(prob(25)) //You either get broken ribs, or a broken limb and IB if you made it this far
+		injury_target = M.get_organ(BODY_ZONE_CHEST)
+		default_damage(M)
+		injury_target.fracture()
+	else
+		injury_target = M.get_organ(pick(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_R_LEG))
+		if(!injury_target)
+			return
+		default_damage(M)
+		injury_target.fracture()
+		injury_target.cause_internal_bleeding()
```

Comment:
`default_damage(M)` here is executed no matter if you hit the 25% probability or not (assuming the guard clause is a partial oversight), and in fact it shows up in nearly every `if` block in this proc - if we basically always want to apply some default damage to the contractee, why not just run it unconditionally, say, at the start of the proc?